tests/controllers/controller_with_response_headers.py

```diff
@@ -0,0 +1,35 @@
+from uuid import UUID
+
+import winter
+from winter import ResponseHeader
+
+
+@winter.controller
+@winter.route('with-response-headers/')
+class ControllerWithResponseHeaders:
+
+    @winter.response_header('x-header', 'header')
```

Comment:
I suggest to move to @winter.http.response_header(...)

Reply:
What about existing winter.request_body then?

Reply:
seems wrong

Reply:
so I remove both request_body and request_header from init.py, leaving them only in http's init.py?

Reply:
what to do with looking similar response_status (now it's not even in http directory)?

Reply:
Also we can move it in http(may be in different pr may be in this pr)